x-pack/plugins/security_solution/public/detection_engine/rule_response_actions/response_actions_form.tsx

```diff
@@ -53,14 +53,21 @@ export const ResponseActionsForm = ({
       const fieldErrors = reduce<string[], Array<{ type: string; errors: string[] }>>(
         map(items, 'path'),
         (acc, path) => {
-          if (fields[`${path}.params`]?.errors?.length) {
-            acc.push({
-              type: upperFirst((fields[`${path}.actionTypeId`].value as string).substring(1)),
-              errors: map(fields[`${path}.params`].errors, 'message'),
-            });
-            return acc;
-          }
+          map(fields, (_, name) => {
+            const paramsPath = `${path}.params`;
+
+            if (name.includes(paramsPath)) {
+              if (fields[name]?.errors?.length) {
+                acc.push({
+                  type: upperFirst((fields[`${path}.actionTypeId`].value as string).substring(1)),
```

Comment:
What are you `upperFirst()` here? and do we have a `const` or `i18n` value for it instead?

Reply:
The values we're getting here are either `.osquery` or `.endpoint` and after substring and upperFirst we get Osquery or Endpoint. I'll create a const for this to make it easier to read. 
However, would you have any other suggestion on how to tackle this?